apps/dine-pleiepenger/src/components/saksbehandlingstid/Saksbehandlingstid.tsx

```diff
@@ -41,18 +41,32 @@ const getFristTekst = (frist: Date, venteårsak?: Venteårsak): React.ReactNode
 const Saksbehandlingstid: React.FunctionComponent<Props> = ({ frist, venteårsak, saksbehandlingstidUker = 7 }) => {
     const fristErPassert = frist ? erSaksbehandlingsfristPassert(frist) : false;
     const { text } = useMessages();
+
+    const getMelding = () => {
```

Comment:
Det er en veldig mikrooptimalisering, men jeg liker å unngå å definere funksjoner og kalle dem i en komponent på denne måten. Du definerer en funksjon som egentlig ikke trengs, på et vis… 🤷🏻 

Det er to forskjellige approaches jeg vanligvis heller velger:
- bare inline logikken som JSX. I dette tilfellet ville det nok blitt en smule for mye nøsting av ternaries.
- lage en støttekomponent. Når man returnerer JSX, er det jo digg å bruke det React har laget for oss for å gjøre nettopp det. Så sleng sammen en `<FristMelding />` komponent som tar inn all info som props og skjuler eventuell kompleksitet (`fristErPassert` for eksempel).

Reply:
Koker vel bare ned til preferanser. Men sånne inline-funksjoner føler jeg er bug-feller